edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/TestServer/Startup.cs

```diff
@@ -19,22 +20,23 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc();
+            services.AddControllers().AddNewtonsoftJson();
```

Comment:
FYR https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-3.1&tabs=visual-studio#newtonsoftjson-jsonnet-support